### PR TITLE
Remove Ubuntu 15.10 (Wily Werewolf) from build-deb targets

### PR DESCRIPTION
**Pull request by mlaventure**
As of July 28, 2016, it is officially EOL (https://lists.ubuntu.com/archives/ubuntu-announce/2016-July/000210.html) and will receive no further updates.

Signed-off-by: Kenfe-Mickael Laventure mickael.laventure@gmail.com

---

bye!

ping @tianon

### CODE REVIEW

**Comment by tianon** [comment]
LGTM! :heart:

**Comment by thaJeztah** [comment]
This will need changes to the installation docs; but those will be moved out soon https://github.com/docker/docker/blob/master/docs/installation/linux/ubuntulinux.md

**Comment by thaJeztah** [comment]
@mlaventure looks like the generate script needs an update as well https://github.com/docker/docker/blob/9ffd2652e249f02515410487bdc41e348e7979b1/contrib/builder/deb/amd64/generate.sh#L84

**Comment by thaJeztah** [comment]
Oh, sorry for the noise, but is this the last distro we have that did _not_ use systemd? Perhaps we can remove some stuff in that area after this

**Comment by mlaventure** [comment]
@thaJeztah we still support `trusty` :)

I'll update the script.

**Comment by mlaventure** [comment]
@tianon should `Dockerfile.aarch64` be updated to use `aarch64/ubuntun:xenial`?

**Comment by thaJeztah** [comment]
> we still support trusty :)

Oh! Silly me. Corner of shame is calling me

**Comment by justincormack** [comment]
@mlaventure yes that would be a good idea. I can test it on aarch64....

**Comment by tianon** [comment]
Indeed, +1 for updating that too! :+1: :innocent:

File: docs/installation/linux/ubuntulinux.md [hunk 1]
```diff
@@ -82,10 +82,6 @@ packages from the new repository:
 
             deb https://apt.dockerproject.org/repo ubuntu-trusty main
 
-    - Ubuntu Wily 15.10
```
Comment (thaJeztah) [inline docs/installation/linux/ubuntulinux.md]: You forgot one at the top;
https://github.com/docker/docker/pull/27042/files#diff-26ed3ddfd9bfa4b4334997b3d9bcad44R18

**Comment by mlaventure** [comment]
Updated.

However, I don't have arm 64bits ready for testing the `Dockerfile.aarch64` though. @tianon would you be able to build it and check that nothing else needs to be changed?

**Comment by justincormack** [comment]
I looked at the aarch64 file and there are a bunch of changes needed eg to Go bootstrap, can do in another PR.

**Comment by thaJeztah** [comment]
> can do in another PR.

Yes, I prefer to keep it separate as well, as they're not directly related, and would only block this PR

**Comment by mlaventure** [comment]
Removed the changed to `Dockerfile.aarch64`

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by tianon** [comment]
Totally fair!  re-LGTM :+1:

**Comment by thaJeztah** [comment]
This needs changes in the docker.github.io repo now (probably)

ping @londoncalling how do we deal with that?

**Comment by londoncalling** [comment]
@thaJeztah I'm figuring it out .. will get to it tonight or tomorrow

**Comment by thaJeztah** [comment]
@londoncalling thanks! I wasn't sure if we should be opening pull requests on that repository already

**Comment by mlaventure** [comment]
ping @londoncalling :)

**Comment by londoncalling** [comment]
@thaJeztah, you are right and I'm going to just manually add them over there for now. Docs will be served from that new repo starting later today, and you all can file PR's against that repo. I'll ping you in a second.

**Comment by londoncalling** [comment]
@thaJeztah @mlaventure @justincormack see https://github.com/docker/docker.github.io/pull/35 for the docs change. You can remove the docs change from this PR, but check/merge other files (code) here.

**Comment by mlaventure** [comment]
reverted docs change

**Comment by londoncalling** [comment]
LGTM

**Comment by justincormack** [comment]
LGTM

**Pull request merged**